messages/pt.json

```diff
@@ -1,17 +1,25 @@
 {
+  "store/checkout-payment.selectedPaymentLabel": "Você selecionou",
   "store/checkout-payment.installmentsButton": "Escolher parcelamento",
+  "store/checkout-payment.creditCardExtraDataMessage": "Para sua segurança, preencha os seguintes dados do cartão de crédito:",
   "store/checkout-payment.documentLabel": "Documento do titular do cartão",
   "store/checkout-payment.requiredField": "Campo obrigatório.",
   "store/checkout-payment.invalidDigits": "Dígitos inválidos",
   "store/checkout-payment.creditCardLabel": "Cartão de Crédito",
-  "store/checkout-payment.installmentOptionLabel": "Escolha uma opção de parcelamento",
+  "store/checkout-payment.viewInstallmentOptionsLabel": "Ver opções de parcelamento",
+  "store/checkout-payment.installmentsOptionsTitle": "Opções de parcelamento",
+  "store/checkout-payment.installmentWithoutValue": "{count}x {hasInterestRate, select, false {sem juros} true {com juros de {interestRate}% ao mês}}",
+  "store/checkout-payment.installmentsTitle": "Parcelamento",
+  "store/checkout-payment.installmentOptionLabel": "Escolha um parcelamento",
   "store/checkout-payment.installmentValue": "{installments}x {value}",
+  "store/checkout-payment.changeInstallments": "Alterar",
```

Comment:
UX issue I guess, but this was ambiguous to me. When I saw the button, I expected it to allow me to change card data.

suggestion: talk to designers to change this to "Alterar parcelamento"
question: editing card data (as I intended to do) is a security breach?

Reply:
> UX issue I guess, but this was ambiguous to me. When I saw the button, I expected it to allow me to change card data.

but the button is right beside the installments info 🤔

> editing card data (as I intended to do) is a security breach?

what do you mean? I don't think I understood

Reply:
> but the button is right beside the installments info 🤔

Concordo que faz sentido, mas quando eu bati o olho, editar o cartão foi a primeira coisa que me veio em mente. Talvez seja porque eu queria realizar essa ação e só tinha um botão, então assumi que era sobre isso. Enfim, não é uma required change, mas acho interessante comentar com os designers.

> what do you mean? I don't think I understood

É porque assim que o usuário termina de preencher os dados do cartão + parcelamento, o "card preview" esconde os últimos dígitos **e** não existe botão para editar o cartão. Então me ficou parecendo que isso é uma questão de segurança

Reply:
O possível problema é que é possível apagar o cartão e ao clicar em "adicionar novo cartão", os dados anteriores ainda aparecem

Reply:
> Concordo que faz sentido, mas quando eu bati o olho, editar o cartão foi a primeira coisa que me veio em mente. Talvez seja porque eu queria realizar essa ação e só tinha um botão, então assumi que era sobre isso. Enfim, não é uma required change, mas acho interessante comentar com os designers.

entendi, vou levantar isso com eles

> É porque assim que o usuário termina de preencher os dados do cartão + parcelamento, o "card preview" esconde os últimos dígitos e não existe botão para editar o cartão. Então me ficou parecendo que isso é uma questão de segurança
>
> O possível problema é que é possível apagar o cartão e ao clicar em "adicionar novo cartão", os dados anteriores ainda aparecem

vou falar pelo slack 👀